src/apis/ConfigApis.js

```diff
@@ -1,20 +1,24 @@
 import axios from 'axios'
-
 import { loadProgressBar } from 'axios-progress-bar'
 
-if (process.env.NODE_ENV === 'development') {
-  axios.defaults.socketUrl = location.protocol + '//' + location.hostname + ':' + process.env.PORT
```

Comment:
i don't know how this bit of code works or really what it was meant for, have you checked if it breaks the `npm run dev` which uses `webpack-dev-server` ?

Reply:
I have tested both dev and prod builds. There is still an issue for the dev version: it seems it can't correctly find the zwave logo. All the rest works as expected.

Reply:
The code - I believe - allowed the dev seerver to correctly point to the websocket path. Right now we are using the namespace + base path combination, so it's no longer needed.

Reply:
IMO the problem is here

Reply:
@robertsLando The requests to `/socket.io` are proxied by the webpack webserver to localhost:8091 -- I tested it yesterday, and as long as you have the services running on that port it should work. Let me check what's going on and fix the lint errors.

Reply:
Did you try to run `npm run dev` and `node start` and go to `localhost:8092`? It doesn't work to me

Reply:
@robertsLando Try now! Forgot to add `/api` and `/health` to the proxied endpoints. See [this part](https://github.com/OpenZWave/Zwave2Mqtt/pull/540/files#diff-c8c5ee8e781f831e4927a392fa3565e6R13) of the configuration.

  This will have the webpack webserver proxy requests done on port 8092 for `/api`, `/socket.io` and `/health` forwarded to `localhost:8091`. I prefer this solution so that you don't have to handle _development_ cases in production code.

  Give this fixed version a shot and let me know what you think.

  Saluti da un Italiano in Iowa! :)

Reply:
Just tried now and everything is working as expected! Both `webpack-dev` and `health` endpoints, love your solution, I admit I'm not a webpack expert and I dind't know about that, thanks for this :) I think everything is ready for merge now